pkgs/servers/x11/xorg/xwayland.nix

```diff
@@ -104,6 +105,12 @@ stdenv.mkDerivation rec {
     (lib.mesonBool "libunwind" (libunwind != null))
```

Comment:
Dreaded nulls should be removed.

Reply:
Can you be a bit more specific? How should it look like?

Reply:
Using null as in

```nix
{
. . .
, libABC ? null
. . .
}:

. . . if (libABC != null)  . . .
```

is a problematic anti-pattern. A recent example of problems with it:
https://github.com/NixOS/nixpkgs/pull/233301#issuecomment-1567661913

My personal recommendation is to use a Boolean guard, like `withLibABC`.

Reply:
But this can be done later. For now the update is OK.

Reply:
Aha, makes sense. Added `withLibunwind` guard and used that instead.